src/Locale/LocaleServiceProvider.php

```diff
@@ -41,19 +40,17 @@ public function register()
         $this->app->alias(LocaleManager::class, 'flarum.localeManager');
 
         $this->app->singleton('translator', function () {
-            $defaultLocale = $this->getDefaultLocale();
-
-            $translator = new Translator($defaultLocale, new MessageSelector());
-            $translator->setFallbackLocales([$defaultLocale, 'en']);
+            $translator = new Translator($this->getDefaultLocale(), new MessageSelector());
+            $translator->setFallbackLocales(['en']);
             $translator->addLoader('prefixed_yaml', new PrefixedYamlFileLoader());
 
             return $translator;
         });
         $this->app->alias('translator', Translator::class);
-        $this->app->alias('translator', TranslatorInterface::class);
+        $this->app->alias('translator', TranslatorContract::class);
     }
 
-    private function getDefaultLocale()
+    private function getDefaultLocale(): string
```

Comment:
A return type! That tickles 😊

Reply:
Wasn't really sure, but since we're adopting php 7.0+ and this method really only returns a string it seemed appropriate. Also because it seems to become an industry standard to get of unnecessary phpdoc's completely when type hinting do the same. What's your opinion or preference?

Reply:
Yes let's go for it! I only meant that it felt funny, but in a good kind of way 😆

Reply:
Done, patched the whole class.